modules/core/migrations/v5/migrations.go

```diff
@@ -0,0 +1,44 @@
+package v5
```

Comment:
looks like leftover meger conflicts? Or maybe needs to be moved to v6 migration?

Reply:
nice catch